relayer/strategies.go

```diff
@@ -22,8 +22,9 @@ type ActiveChannel struct {
 }
 
 const (
-	ProcessorEvents string = "events"
-	ProcessorLegacy        = "legacy"
+	ProcessorEvents              string = "events"
+	ProcessorLegacy                     = "legacy"
+	defaultClientUpdateThreshold        = 6 * time.Hour
```

Comment:
In combination with this suggestion https://github.com/cosmos/relayer/pull/1008/files#r1000932480, I think we can update this to `0*time.Millsecond`. We should use this variable for the flag default also